src/tiny.js

```diff
@@ -1,16 +1,23 @@
 /* eslint no-unused-vars:0 */
 import createGlamorous from './create-glamorous'
 
-function splitProps({
-  css: cssProp,
-  // these are plucked off
-  theme, // because they
-  className, // should never
-  innerRef, // be forwarded
-  glam, // to the lower
-  // component ever
-  ...rest
-}) {
+function splitProps(
+  {
+    css: cssProp,
+    // these are plucked off
+    theme, // because they
+    className, // should never
+    innerRef, // be forwarded
```

Comment:
comments here are not relevant to `innerRef` any more. I'd move the prop above the comments.

Reply:
👍